BASE/src/ServerTelemetryChannel/ServerTelemetryChannel.cs

```diff
@@ -121,6 +121,22 @@ public string EndpointAddress
             set { this.TelemetrySerializer.EndpointAddress = new Uri(value); }
         }
 
+        /// <summary>
+        /// Gets or Sets the subscriber to an event in Transmission to track ingestion status metrics.
```

Comment:
The comment is bit misleading - there is nothing about metrics here. Registered callback gets called with HttpResponse  - whether its used for metrics extraction or something else - this module doesn't care.

Reply:
Fixed in both `ServerTelemetryChannel` and `TelemetrySerializer`.